mlperf_logging/mllog/constants.py

```diff
@@ -167,7 +167,7 @@
 EVAL_MAX_PREDICTION_SYMBOLS = "eval_max_prediction_symbols"
 START_WARMUP_STEP = "start_warmup_step"
 INIT_CHECKPOINT_STEP = "init_checkpoint_step"
-
+LORA_ALPHA = "lora_alpha"
```

Comment:
I can't really comment in in the right place, but in L40 there are benchmark-specific name constants. Lora should go there as well

Reply:
in L40 we have benchmark names. Alpha is a hyperparameter. Anyway I added  in L40
LLAMA2_70B_LORA = "llama2_70b_lora"